metrics/repl/repl.go

```diff
@@ -86,9 +90,6 @@ func (c *Repl) Help() blip.CollectorHelp {
 	}
 }
 
-var statusQuery = "SHOW SLAVE STATUS" // SHOW REPLICA STATUS as of 8.022
```

Comment:
Please put that comment back, i.e. (few lines up in the diff):
```go
		statusQuery:     "SHOW SLAVE STATUS",  // SHOW REPLICA STATUS as of 8.022
```

When working with MySQL, you'll find that such "as of <version>" comments are common and helpful, especially when your career with MySQL begins to span decades. Lots and lots of changes, and who can remember them all?